File: 3st/Juhee/외계행성의 나이.java
class Solution {
    public String solution(int age) {
        String[] str = {"a", "b", "c", "d", "e", "f", "g", "h", "i", "j"};
        StringBuilder answer = new StringBuilder();

        while (age > 0) {
            int digit = age % 10;
            answer.insert(0, str[digit]);
            age /= 10;
        }

        return answer.toString();
    }
}

File: 3st/Juram/programmer/외계행성의 나이.js
function solution(age) {
    const map = {
        "0": "a", "1": "b", "2": "c", "3": "d", "4": "e",
        "5": "f", "6": "g", "7": "h", "8": "i", "9": "j"
    };
     return age.toString().split("").map(digit => map[digit]).join("");
}